docs/spec/sanitations.md

```diff
@@ -8,7 +8,7 @@ _Edition_: Swan Lake
 The following command was used to generate the Ballerina client from the OpenAPI specification. The command should be executed from the repository root directory.
 
 ```bash
-$ bal openapi -i docs/spec/openapi.yml --mode client --license docs/license.txt -o ballerina
+$ bal openapi -i docs/spec/openapi.yml --mode client --license docs/license.txt -o ballerina --use-sanitized-oas
```

Comment:
This is an experimental feature and does not related to sanitizations done manually. Let's not add it here.

```suggestion
$ bal openapi -i docs/spec/openapi.yml --mode client --license docs/license.txt -o ballerina
```